opal/mca/threads/qthreads/threads_qthreads.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2004-2005 The Trustees of Indiana University and Indiana
+ *                         University Research and Technology
+ *                         Corporation.  All rights reserved.
+ * Copyright (c) 2004-2014 The University of Tennessee and The University
+ *                         of Tennessee Research Foundation.  All rights
+ *                         reserved.
+ * Copyright (c) 2004-2005 High Performance Computing Center Stuttgart,
+ *                         University of Stuttgart.  All rights reserved.
+ * Copyright (c) 2004-2005 The Regents of the University of California.
+ *                         All rights reserved.
+ * $COPYRIGHT$
```

Comment:
I don't really care, but I'll point out that you're adding a new file with a copyright block in it that dates back to 2005.  It does not have your copyright in it, or a copyright block that says 2019 or 2020.  That seems... odd.

Reply:
think this is reslved.